### ISSUE TITLE
[BUG] rlp.encode([]) throws invalid type Error in the Truffle console

### ISSUE THREAD

**Issue by wanseob** [ethereumjs/rlp#69]
**Describe the bug**
Encoding an array does not work in truffle console while it works in the node console.

**To Reproduce**
```bash
$ mkdir testDir && cd testDir
$ truffle init
$ npm install --save-dev rlp
$ node
> const rlp = require('rlp')
> rlp.encode()
<Buffer 80>
> rlp.encode([])
<Buffer c0>
> .exit
$ truffle develop
trufle(develop)> const rlp = require('rlp')
trufle(develop)> rlp.encode([])
<Buffer 80>
trufle(develop)> rlp.encode([])
Error: invalid type
    at toBuffer (/home/james/Temp/test/node_modules/rlp/dist/index.js:232:19)
    at Object.encode (/home/james/Temp/test/node_modules/rlp/dist/index.js:20:24)
...
```
**Expected behavior**
Encode array without any error also in the truffle console

**Screenshots**
![image](https://user-images.githubusercontent.com/8542397/51656024-adc5c480-1fe2-11e9-9e4b-c8eb863afa70.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu18.04
 - zsh: v5.4.2
 - Node: v10.15.0
 - Truffle: v5.0.2
 - rlp: v2.2.2

**Comment by cag** [comment]
@wanseob PRs addressing this have been merged, but are not in an official release yet. If you want to use this fix before it gets published officially and you're using NPM, install my fork branch `github:cag/rlp#array-fix-with-tests`.

**Comment by holgerd77** [comment]
I can do a short-term release, didn't know this was urgent. Will do during next week.

**Comment by wanseob** [comment]
I really love this library. Thanks for your work :+1:

**Comment by wanseob** [comment]
Closes this issue because #70 being included in the version v2.2.3 #73
